### ISSUE TITLE
New Shared Parameters File. Create data but not the correct Datatype

### ISSUE THREAD

**Issue by icarden** [erfajo/OrchidForDynamo#175]
### Operating system
Windows 10 Pro for Workstations

### Revit and Dynamo version
Revit: 22.1.30.34
Dynamo: 2.12.1.8246
Orchid: 212.3.0.8347

### What did you do? 
Create a new Shared parameters *.txt file (Revit in German)

### What did you expect to see?
to keep the parameters in the correct type (Text instead of Zahl)

### What did you see instead?
All new parameters were placed in the Zahl (Currency) Category.

![2022-11-08 17_41_21-Dynamo](https://user-images.githubusercontent.com/53174781/200624784-84b575de-97b0-4d8d-b58c-9a08f7b3c64c.jpg)

**Comment by erfajo** [comment]
Please add you graph file here. You can add .txt to the extension and then upload it.
However, why is the Document.Current being yellow? do you use the proper Document node for families?

**Comment by icarden** [comment]
Hi erfajo,
Indeed, the problem is strange. I attach the solution I used (it is not the most optimal). Previously for other projects, this solution worked without any problem. The file is used directly in a family. I also attach the excel file in case there is a typing problem.

[GPHH_GGParametern_Test_.dyn.txt](https://github.com/erfajo/OrchidForDynamo/files/9979370/GPHH_GGParametern_Test_.dyn.txt)
[H+P_GGP_Hauptparametern_A.xlsx](https://github.com/erfajo/OrchidForDynamo/files/9979373/H%2BP_GGP_Hauptparametern_A.xlsx)

**Comment by erfajo** [comment]
Sorry but yoy have not added all  needed input to the SharedParameter.Create node, without doing that I do not need to look any further. That will never go and this goes for any dynamo node. If it dont get the demanded input, noting will happen as could be expected.

**Comment by erfajo** [comment]
First, do never add anything to an issue that is off-topic, this is cluttering anything. However, I see you as a newbie who doesn’t know how to act, and therefore do I take the time to answer, under any other circumstance would your post be deleted!
Do always create a new issues instead of addind to others issues if you don’t have EXACTLY the same problem. The problem in this issue is about missing inputs, yours is understanding the node and Dynamo in general!?

Next, if you need general support then go to https://forum.dynamobim.com

Lastly, The thing with dynamo is to see what a given node takes as the input, and this goes for every single input! Does it take single items, lists, or do the node have a default input you can accept. Then try to see how you can provide this input. You might have a case where you feed the input with a list but it should be a single item. In those cases is “list level” invented. This can furthermore be extended with “lacing”.
In other words, try to see what the inports in the node takes. Then try to see if you can provide that structure from your excel sheet, either by having a sheet with columns that represents the needed input, for every single line or try to see what can be grouped somehow.

**Comment by erfajo** [comment]
@icarden
I am closing this issue due to cluttering from another user, If something should be a bug, then please return.
